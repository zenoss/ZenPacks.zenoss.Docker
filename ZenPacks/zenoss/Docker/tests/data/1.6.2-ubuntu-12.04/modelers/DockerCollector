Docker version 1.6.2, build 7c8fca2
__SPLIT__
CONTAINER ID                                                       IMAGE                           COMMAND                                          CREATED             STATUS                   PORTS                                                 NAMES
8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa   nginx:latest                    "nginx -g 'daemon off;'"                         3 hours ago         Up 3 hours               443/tcp, 0.0.0.0:8080->80/tcp, 0.0.0.0:8081->81/tcp   nginx               
9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f   dkuffner/docker-stress:latest   "stress --cpu 8 --io 4 --vm 2 --vm-bytes 128M"   3 hours ago         Up 3 hours                                                                     desperate_babbage   
46ff00ea5094ffd48c5f1bbce0799e2ffce97554fdb086a8dfb8dac7b0f9fac3   hello-world:latest              "/hello"                                         3 hours ago         Exited (0) 3 hours ago                                                         determined_morse    
__SPLIT__
18 23 0:17 / /sys rw,nosuid,nodev,noexec,relatime shared:7 - sysfs sysfs rw
19 23 0:4 / /proc rw,nosuid,nodev,noexec,relatime shared:12 - proc proc rw
20 23 0:6 / /dev rw,nosuid,relatime shared:2 - devtmpfs udev rw,size=3819384k,nr_inodes=954846,mode=755
21 20 0:14 / /dev/pts rw,nosuid,noexec,relatime shared:3 - devpts devpts rw,gid=5,mode=620,ptmxmode=000
22 23 0:18 / /run rw,nosuid,noexec,relatime shared:5 - tmpfs tmpfs rw,size=765728k,mode=755
23 0 202:1 / / rw,relatime shared:1 - ext4 /dev/xvda1 rw,discard,data=ordered
24 18 0:12 / /sys/kernel/security rw,nosuid,nodev,noexec,relatime shared:8 - securityfs securityfs rw
25 20 0:19 / /dev/shm rw,nosuid,nodev shared:4 - tmpfs tmpfs rw
26 22 0:20 / /run/lock rw,nosuid,nodev,noexec,relatime shared:6 - tmpfs tmpfs rw,size=5120k
27 18 0:21 / /sys/fs/cgroup ro,nosuid,nodev,noexec shared:9 - tmpfs tmpfs ro,mode=755
28 27 0:22 / /sys/fs/cgroup/systemd rw,nosuid,nodev,noexec,relatime shared:10 - cgroup cgroup rw,xattr,release_agent=/lib/systemd/systemd-cgroups-agent,name=systemd
29 18 0:23 / /sys/fs/pstore rw,nosuid,nodev,noexec,relatime shared:11 - pstore pstore rw
30 27 0:24 / /sys/fs/cgroup/cpuset rw,nosuid,nodev,noexec,relatime shared:13 - cgroup cgroup rw,cpuset
31 27 0:25 / /sys/fs/cgroup/memory rw,nosuid,nodev,noexec,relatime shared:14 - cgroup cgroup rw,memory
32 27 0:26 / /sys/fs/cgroup/devices rw,nosuid,nodev,noexec,relatime shared:15 - cgroup cgroup rw,devices
33 27 0:27 / /sys/fs/cgroup/freezer rw,nosuid,nodev,noexec,relatime shared:16 - cgroup cgroup rw,freezer
34 27 0:28 / /sys/fs/cgroup/net_cls,net_prio rw,nosuid,nodev,noexec,relatime shared:17 - cgroup cgroup rw,net_cls,net_prio
35 27 0:29 / /sys/fs/cgroup/hugetlb rw,nosuid,nodev,noexec,relatime shared:18 - cgroup cgroup rw,hugetlb
36 27 0:30 / /sys/fs/cgroup/cpu,cpuacct rw,nosuid,nodev,noexec,relatime shared:19 - cgroup cgroup rw,cpu,cpuacct
37 27 0:31 / /sys/fs/cgroup/perf_event rw,nosuid,nodev,noexec,relatime shared:20 - cgroup cgroup rw,perf_event
38 27 0:32 / /sys/fs/cgroup/blkio rw,nosuid,nodev,noexec,relatime shared:21 - cgroup cgroup rw,blkio
39 27 0:33 / /sys/fs/cgroup/pids rw,nosuid,nodev,noexec,relatime shared:22 - cgroup cgroup rw,pidst
