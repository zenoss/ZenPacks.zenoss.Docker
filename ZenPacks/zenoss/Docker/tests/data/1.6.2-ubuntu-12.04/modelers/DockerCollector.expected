# vim: ft=yaml
command: "docker -v 2>/dev/null ; echo __SPLIT__ ; sudo docker ps -a --no-trunc 2>/dev/null ; echo __SPLIT__ ; cat /proc/self/mountinfo 2>/dev/null"

expected:
  - docker_version: "Docker version 1.6.2, build 7c8fca2"

  - relname: "docker_containers"
    maps:
      - id: "8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa"
        modname: "ZenPacks.zenoss.Docker.DockerContainer"
        title: "nginx"
        image: "nginx:latest"
        command: "\"nginx -g 'daemon off;'\""
        created: "3 hours ago"
        ports: "443/tcp, 0.0.0.0:8080->80/tcp, 0.0.0.0:8081->81/tcp"
        cgroup_path: "/sys/fs/cgroup"

      - id: "9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f"
        modname: "ZenPacks.zenoss.Docker.DockerContainer"
        title: "desperate_babbage"
        image: "dkuffner/docker-stress:latest"
        command: "\"stress --cpu 8 --io 4 --vm 2 --vm-bytes 128M\""
        created: "3 hours ago"
        ports: ""
        cgroup_path: "/sys/fs/cgroup"

      - id: "46ff00ea5094ffd48c5f1bbce0799e2ffce97554fdb086a8dfb8dac7b0f9fac3"
        modname: "ZenPacks.zenoss.Docker.DockerContainer"
        title: "determined_morse"
        image: "hello-world:latest"
        command: "\"/hello\""
        created: "3 hours ago"
        ports: ""
        cgroup_path: "/sys/fs/cgroup"
